packages/vuetify/src/components/VSelect/VSelect.tsx

```diff
@@ -215,7 +217,7 @@ export const VSelect = genericComponent<new <
             },
           ]}
           appendInnerIcon={ props.menuIcon }
-          readonly
+          readonly={ props.readonly }
```

Comment:
Should never be able to type in here

Reply:
Ok now it make sense why it was always `true`, didnt quite thought about that. Thought it was just missed.